test/e2e/run.sh

```diff
@@ -135,7 +136,11 @@ screen-install-cri-resmgr-debugging() {
 
 screen-launch-cri-resmgr() {
     speed=60 out "### Launching cri-resmgr with config $cri_resmgr_cfg."
-    launch cri-resmgr
+    if [ "$binsrc" == "packages/debian-10" ]; then
```

Comment:
Same comment here as earlier, I'd rather use a catch-all case for packaging tests:
`[ "${binsrc#packaging/}" != "$binsrc" ]`...

Reply:
Changed this to matching all package installations. It'll work as long as rpm's and deb's all install systemd service.